### PR TITLE
Hometown labeling

### PR DESCRIPTION
**Pull request by JohnPMurray**
Updated the CSV to contain a field for both hometown and display name. I'm pretty sure the labeling comes from the CSV that gets generated, but we might want to double check on that.

I've already merged in the changes from build-error-hotfix into this branch, so we should probably merge that one first.

Size S:
As an Admin I want student hometowns to appear in my generating labeling file so I can create labels with the hometowns included.

[https://trello.com/c/EW5iJNS1/82-admin-student-hometown-labeling](https://trello.com/c/EW5iJNS1/82-admin-student-hometown-labeling)

### CODE REVIEW

File: backend/routes/download.js [hunk 1]
```diff
@@ -240,6 +240,8 @@ router.route('/csv/:showId')
                       studentEmail: `${entryData.studentUsername}@rit.edu`,
```
Comment (JohnPMurray) [inline backend/routes/download.js:240]: This is the only file that was actually changed, the rest of the PR is just the merged changes from build-error-hotfix

File: backend/routes/download.js [hunk 2]
```diff
@@ -240,6 +240,8 @@ router.route('/csv/:showId')
...
                       studentFirstName: user ? user.firstName : null,
                       studentLastName: user ? user.lastName : null,
+                      studentHometown: user ? user.hometown : null,
+                      studentDisplayName: user ? user.displayName : null,
```
Comment (JohnPMurray) [inline backend/routes/download.js:244]: The models already contain hometown and displayname, we just need to tell the software to grab that data for the CSV

File: backend/routes/download.js [hunk 3]
```diff
@@ -299,6 +301,8 @@ router.route('/csv/:showId')
                   studentEmail: 'Student Email',
                   studentFirstName: 'Student First Name',
                   studentLastName: 'Student Last Name',
+                  studentHometown: 'Student Homewtown',
+                  studentDisplayName: 'Student Display Name',
```
Comment (JohnPMurray) [inline backend/routes/download.js:305]: Titles for CSV columns

**Review by johnbyrne727rit — APPROVED** [review]
Looks good to me.

File: backend/routes/download.js (hunk 3 above)
Comment (johnbyrne727rit) [reply backend/routes/download.js:305]: Makes sense

**Review by moiseslorap — APPROVED** [review]

**Pull request merged**